fem/bilininteg_vecmass.cpp

```diff
@@ -30,6 +31,16 @@ void VectorMassIntegrator::AssemblePA(const FiniteElementSpace &fes)
    ElementTransformation *T = mesh->GetElementTransformation(0);
    const IntegrationRule *ir
       = IntRule ? IntRule : &MassIntegrator::GetRule(el, el, *T);
+#ifdef MFEM_USE_CEED
+   if (DeviceCanUseCeed())
+   {
+      if (ceedDataPtr) { delete ceedDataPtr; }
+      CeedData* ptr = new CeedData();
+      ceedDataPtr = ptr;
+      InitCeedCoeff(Q, ptr);
+      return CeedPAMassAssemble(fes, *ir, *ptr);
```

Comment:
```suggestion
      delete ceedDataPtr;
      ceedDataPtr = new CeedData;
      InitCeedCoeff(Q, ceedDataPtr);
      return CeedPAMassAssemble(fes, *ir, *ceedDataPtr);
```

Same changes also apply in `bilininteg_mass_pa.cpp` and `bilininteg_diffusion_pa.cpp`.